src/index.html

```diff
@@ -1,12 +1,302 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" >
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="logo">
+          <a href="#home" class="logo__link">
+            <img src="./images/logo.png"
+            alt="logo"
+            class="logo__img"
+            >
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#laptops&computers"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link">Gadgets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link">Tablets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#photo" class="nav__link">Photo</a>
+            </li>
+            <li class="nav__item">
+              <a href="#video" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="container-card">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__photo">
```

Comment:
Fix all images format
```suggestion
        <img
          src="./images/imac.jpeg"
          alt="imac"
          class="card__photo"
        >
```